engines/dgds/console.cpp

```diff
@@ -284,4 +286,191 @@ bool Console::cmdScene(int argc, const char **argv) {
 	return true;
 }
 
+void Console::printOp(int indent, const char *text) {
+	for (int i = 0; i < indent; i++) {
+		debugPrintf("\t");
+	}
+	debugPrintf("%s", text);
+}
+
+bool Console::cmdScriptDump(int argc, const char **argv) {
+	if (argc < 2) {
+		debugPrintf("Usage: %s <filename>\n", argv[0]);
+		debugPrintf("  eg: %s CLBATH.ADS\n", argv[0]);
+		return true;
+	}
+
+	Common::String fname(argv[1]);
+	if (!_vm->getResourceManager()->hasResource(fname))  {
+		debugPrintf("Resource %s not found\n", fname.c_str());
+		return true;
+	}
```

Comment:
You don't need this check, since you're effectively re-checking for file existence below as well

Reply:
Thank you, the redundant check was removed